frontend/src/global_styles/content/_projects_list.sass

```diff
@@ -134,3 +137,14 @@ body.controller-projects.action-index
     padding-left: 1em
   li
     list-style-type: none
+
+.project-list-container
+  display: flex
+  flex-direction: column
+  max-height: calc(100vh - #{$toolbar-height})
+
+.project-list
```

Comment:
Please move that block as a child of the container, as it does not work without it